docs/code_ref/coordinates.rst

```diff
@@ -142,7 +142,7 @@ It is also possible to transform to any coordinate system implemented in Astropy
     >>> greenbelt_frame = AltAz(obstime=time, location=greenbelt)
     >>> west_limb = SkyCoord(900*u.arcsec, 0*u.arcsec, frame=frames.Helioprojective, obstime=time)
     >>> west_limb.transform_to(greenbelt_frame)  # doctest: +REMOTE_DATA +ELLIPSIS
-    ...
+    Downloading http://maia.usno.navy.mil/ser7/finals2000A.all [Done]
```

Comment:
This is going to fail on a second run, i.e. if you run it locally. Can we just skip this test?

Reply:
fine

Reply:
HAPPY NOW?!

Reply:
I KNOW I AINT

Reply:
GRUMBE